src/Components/ExperimentOne/ExperimentOne.js

```diff
@@ -1,12 +1,21 @@
+import ExperimentOneDisplayInformation from "./ExperimentOneDisplayInformation.js";
 import ExperimentOneHeader from "./ExperimentOneHeader.js";
-import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
 
 const ExperimentOne = (props) => {
+  const { isBackButtonClicked } = props;
```

Comment:
same re: destructuring in function arguments

Reply:
For some reaon I get an error saying `Expected onClick listener to be a function, instead got type object`
I will investigate why is that the case

Reply:
done